tests/foreman/smoke/test_api_smoke.py

```diff
@@ -227,6 +229,7 @@
         u'/api/usergroups/:usergroup_id/external_usergroups',
         u'/api/usergroups/:usergroup_id/external_usergroups/:id',
         u'/api/usergroups/:usergroup_id/external_usergroups/:id',
+        u'/api/usergroups/:usergroup_id/external_usergroups/:id',
```

Comment:
Correct. You can perform multiple actions on a single URL. For example, you might be able to GET, PUT and DELETE a single external usergroup, and all three of those actions are listed separately.